src/ion/moves.rs

```diff
@@ -563,11 +563,10 @@ impl<'a, F: Function> Env<'a, F> {
                     let inst = usedata.pos.inst();
                     let slot = usedata.slot;
                     let operand = usedata.operand;
-                    // Safepoints add virtual uses with no slots;
-                    // avoid these.
-                    if slot != SLOT_NONE {
-                        self.set_alloc(inst, slot as usize, alloc);
-                    }
+
+                    debug_assert_ne!(slot, SLOT_NONE);
```

Comment:
I suspect `SLOT_NONE` may now also be unused (the only point where I can see it used to create a `Use` is in the safepoint-handling loop in liveranges.rs, now deleted) -- can you remove if so?

Reply:
Thanks for catching this!